### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by culpgrant** [Snowflake-Labs/terraform-provider-snowflake#1724] [General] [answered]
Hey, @sfc-gh-swinkler were you planning on doing a release of all the new features this week? Just trying to coordinate the timing on my end! Thank you

**Comment by sfc-gh-swinkler** [comment, marked as answer]
@culpgrant i would like to get this PR merged: https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/1725 and then i will do a release. Plan for tomorrow.

**Reply by culpgrant** [reply]
Awesome thank you! And thanks for maintaining this repo. It has immensely improved recently and I have a lot more confidence in relying on it.

**Reply by lachniej** [reply]
+1 to @culpgrants comment. Big thanks! Can't wait for the cdktf snowflake provider to pick up the next release with on_all parameters for object grants!

**Reply by sfc-gh-swinkler** [reply]
@culpgrant thank you! We are working hard to improve stability.
